run-directive/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Directive
```

Comment:
Nit - should we clarify that this is an `npm directive` in the action name/file?